### PR TITLE
Fix Email resent content

### PR DESCRIPTION
**Pull request by anikbrazeau**
Minor copy tweaks for clarity on screenreader on Email resent page:
https://notification.alpha.canada.ca/email-not-received

Fixes https://github.com/cds-snc/notification-api/issues/967

### CODE REVIEW

**Comment by anikbrazeau** [comment]
Fixes https://github.com/cds-snc/notification-api/issues/967

**Comment by anikbrazeau** [comment]
@brdunfield I found a few more instances with "resend" and fixed those too, tests passing locally 🤞 if you can give it another quick glance before I merge.

File: app/templates/views/notifications/notification.html [hunk 1]
```diff
@@ -49,7 +49,7 @@
         </p>
       {% elif notification_status == 'technical-failure' %}
         <p class="notification-status-cancelled">
-          Technical failure – Notification will resend once the team have
+          Technical failure – GC Notify will re-send once the team have
```
Comment (brdunfield) [inline app/templates/views/notifications/notification.html:52]: I don't see this string in the csvs below.

Comment (brdunfield) [reply app/templates/views/notifications/notification.html:52]: Oh, I guess this isn't even wrapped for translation!

Comment (brdunfield) [reply app/templates/views/notifications/notification.html:52]: We should probably wrap this and add translations to the csvs :)

Comment (anikbrazeau) [reply app/templates/views/notifications/notification.html]: Good call!

**Pull request merged**